app/reducers/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const defaultState = {
-  dats: [],
```

Comment:
I don't think this can simply be changed into an object. `dats` is used as an array to show the list.

Reply:
it's used as an object in the reducer (indexed by key if I'm not mistaken). This looked like a typo to me but I'll investigate further!